library/farmosUtil/farmosUtil.soilDisturbanceTerminationLog.unit.cy.js

```diff
@@ -0,0 +1,630 @@
+import * as farmosUtil from './farmosUtil';
+
+describe('Test the soil disturbance termination log functions', () => {
+  let fieldMap = null;
+  let bedMap = null;
+  let categoryMap = null;
+
+  function createPlantAsset(name, type, timestamp) {
```

Comment:
I would suggest using `date` instead of `timestamp` as the parameter names since you are passing date strings rather than unix timestamps.